EarlGrey/Matcher/GREYMatchers.m

```diff
@@ -438,9 +438,16 @@ @implementation GREYMatchers
       return NO;
     }
     NSInteger row = [element selectedRowInComponent:column];
-    NSString *rowLabel = [element.delegate pickerView:element
-                                          titleForRow:row
-                                         forComponent:column];
+    NSString *rowLabel;
+    if ([element.delegate respondsToSelector:@selector(pickerView:attributedTitleForRow:forComponent:)]) {
+        rowLabel = [element.delegate pickerView:element
```

Comment:
Spacing is fixed. When you say fall back, you mean that if rowLabel is nil, then the method should just return, or return NO?